Darwin/Services/StitchRemoteMongoDBService/StitchRemoteMongoDBServiceTests/SyncPerformanceIntTests.swift

```diff
@@ -0,0 +1,95 @@
+import XCTest
+@testable import MongoSwift
+import StitchCore
+import StitchCoreSDK
+import StitchCoreAdminClient
+import StitchDarwinCoreTestUtils
+@testable import StitchCoreRemoteMongoDBService
+import StitchCoreLocalMongoDBService
+@testable import StitchRemoteMongoDBService
+
+import Foundation
+
+class SyncPerformanceIntTests: BaseStitchIntTestCocoaTouch {
+    let harness = SyncPerformanceIntTestHarness()
+    static let runId = ObjectId()
+    let joiner = ThrowingCallbackJoiner()
+
+    /* func testFailure() {
```

Comment:
[q] why is this commented out?

Reply:
Because this is used to test failure and I didn't want to delete it but it would theoretically break evergreen if evergreen was running these tests properly

Reply:
Alright so add a comment explaining that